docs/user/index.md

```diff
@@ -0,0 +1,133 @@
+
+## Introduction
+
+Perseo is an [Esper-based](http://www.espertech.com/esper/) Complex Event Processing (CEP) software designed to be fully
+_NGSI-v2_-compliant. It uses NGSI-v2 as the communication protocol for events, and thus, Perseo is able to seamless and
+jointly work with _context brokers_. The context broker tested with Perseo and officially supported is
+[Orion Context Broker](https://github.com/telefonicaid/fiware-orion).
+
+Perseo follows a straightforward idea: listening to events coming from context information to identify patterns
+described by rules, in order to immediately react upon them by triggering actions.
+
+By leveraging on the
+[notifications mechanism](http://fiware-orion.readthedocs.io/en/latest/user/walkthrough_apiv2/index.html#subscriptions),
+clients instruct Orion CB to notify Perseo of the changes in the entities they care about (`Event API`). Details of this
+process are explained in the [Orion Subscription part of the User Manual](user/index.md#orion-subscription). Then, rules
+to the CORE Rule Engine can be easily managed using the publicly available
+[WireCloud](https://github.com/Wirecloud/wirecloud)) operational dashboard, or making use of any of the REST clients
+able to programmaticly use the Perseo's `Rule API`. These rules will identify patterns that will trigger actions with
+Orion to create or update entities, or with other different components or external systems, such as Web (HTTP), Email
+(SMTP) or SMS (SMPP) servers.
+
+Perseo allows you to create/edit/delete rules through its API. You can find more accurate information about the Perseo
+API [here](../API/api.md)
+
+## Perseo Rules
+
+Perseo rules follow a simple JSON structure made up of three mandatory _key-value_ fields: **`name`**, **`text`**, and
+**`action`**. The structure of these rules is sketched in the following JSON code:
+
+```json
+{
+   "name":"<The name of the rule>",
+   "text":"<Insert here a valid EPL statement>",
+   "action":{
+      "type":"[update|sms|email|post|twitter]",
+      "parameters":{
+         ...
+      }
+   }
+}
+```
+
+The `name` field refers to the name of the rule, and it is used as rule identifier. It must start by a letter, and can
+contain digits (0-9), underscores (\_) and dashes (-). Their maximum length is set to 50 characters.
+
+The `action` field states the action to be performed by Perseo when the rule triggers. You can also use an **array** of
+action objects if needed. Each of those actions will be executed when the rule is fired, avoiding to duplicate a rule
+only for getting several actions executed.
+
+Last but no least, the `text` field contains the valid EPL statement to be send to the Esper-based core rule engine. The
+value of this field must follow the EPL syntax.
+
+Following there is an example of a valid EPL simple clause ready to be used with Perseo:
+
+```text
+select *, ev.BloodPressure? as Pressure, ev.id? as Meter
+from pattern
+     [every ev=iotEvent(type="BloodMeter")]
+```
```

Comment:
Better use a rule like:
```
select *, bloodPressure? as Pressure
from iotEvent
where (cast(cast(bloodPressure?,String),double)>1.5 and type="BloodMeter")]
```
since rules based in pattern are not really useful

Reply:
Fixed in https://github.com/telefonicaid/perseo-fe/pull/592/commits/d22cc06af352a8a2836bb13125f54528583857eb